IDE/Espressif/ESP-IDF/examples/template/components/wolfssl/include/user_settings.h

```diff
@@ -1058,8 +1058,9 @@ Turn on timer debugging (used when CPU cycles not available)
             #error "USE_CERT_BUFFERS_1024 is already defined. Pick one."
         #endif
 
-        /* Be sure to include in app when using example certs: */
-        #include <wolfssl/certs_test.h>
+        /* Be sure to include in app, not here, when using example certs: */
```

Comment:
Could you just gate it on the USE_CERT_BUFFERS_ macros? Oddly you are defining that below, so maybe you should make sure its included after since the certs_test.h gates based on `USE_CERT_BUFFERS_256` for example?

Reply:
Someone should fix/refactor wolfSSH to not use names that will conflict...

Reply:
They are not the same keys and in different repos. The filename isn't an issue since its included with wolfssl/certs_test.h or wolfssh/certs_test.h, however the name of the variables should be unique.